src/renderer/template.ts

```diff
@@ -28,10 +34,22 @@ export async function renderTemplate(filepath: string, context: TemplateContext)
  * @private
  * @param filepath to import
  */
-function importComponent(filepath: string, context: TemplateContext): Promise<React.ReactElement> {
-  return new Promise((resolve) => {
-    import(filepath).then(component => { resolve(component.default !== undefined ? component.default(context) : undefined) })
-  })
+function importComponent(filepath: string, context: TemplateContext): Promise<React.ReactElement | undefined> {
+  return new Promise((resolve, reject) => {
+    try {
+      // we should import component only in NodeJS
+      if (require === undefined) resolve(undefined);
+      // remove from cache imported file
+      delete require.cache[require.resolve(filepath)];
+
+      const component = require(filepath);
```

Comment:
Whats the reason for switching to require rather then import as previously used?

Reply:
I guess in the end it gets transpiled to the same anyway

Reply:
require has `cache` property with cached imports, `import` probably not, when I searched for solution to resolve the issue, I only found tutorials/suggestions using require.